packages/@ourworldindata/grapher/src/scatterCharts/ScatterPlotChart.tsx

```diff
@@ -809,40 +784,98 @@ export class ScatterPlotChart
                         </g>
                     </>
                 )}
-                {target && (
-                    <Tooltip
-                        id="scatterTooltip"
-                        tooltipManager={this.manager}
-                        x={position.x}
-                        y={position.y}
-                        offsetX={20}
-                        offsetY={-16}
-                        style={{ maxWidth: "250px" }}
-                        title={target.series.label}
-                        subtitle={timeLabel}
-                        dissolve={fading}
-                        notice={targetNotice}
-                    >
-                        <TooltipValueRange
-                            column={this.xColumn}
-                            values={values.map((v) => v.x)}
-                            notice={xNotice}
-                        />
-                        <TooltipValueRange
-                            column={this.yColumn}
-                            values={values.map((v) => v.y)}
-                            notice={yNotice}
-                        />
-                        <TooltipValueRange
-                            column={this.sizeColumn}
-                            values={compact(values.map((v) => v.size))}
-                        />
-                    </Tooltip>
-                )}
+                {this.tooltip}
             </g>
         )
     }
 
+    @computed get tooltip(): JSX.Element | null {
+        if (!this.tooltipState.target) return null
+
+        const {
+            xColumn,
+            yColumn,
+            tooltipState: { target, position, fading },
+        } = this
+        const points = target.series.points ?? []
+        const values = compact(uniq([first(points), last(points)]))
+
+        let { startTime, endTime } = this.manager
+        const { x: xStart, y: yStart } = first(values)?.time ?? {},
+            { x: xEnd, y: yEnd } = last(values)?.time ?? {}
+
+        let xValues = values.map((v) => v.x),
+            xNoticeNeeded =
+                (xStart !== undefined && xStart != startTime && xStart) ||
+                (xEnd !== undefined && xEnd != endTime && xEnd),
+            xNotice = xNoticeNeeded ? [xStart, xEnd] : []
+
+        let yValues = values.map((v) => v.y),
+            yNoticeNeeded =
+                (yStart !== undefined && yStart != startTime && yStart) ||
+                (yEnd !== undefined && yEnd != endTime && yEnd),
+            yNotice = yNoticeNeeded ? [yStart, yEnd] : []
+
+        // handle the special case where the same variable is used for both axes
+        // with a different year's value on each
+        if (xColumn.def.datasetId === yColumn.def.datasetId) {
+            const xTime = xColumn.def.targetTime ?? this.manager.endTime,
+                yTime = yColumn.def.targetTime ?? this.manager.endTime
```

Comment:
There's the (maybe theoretical) scenario where the chart has a timeline, which then also allows for a multi-year timeline. The chart we get out of that looks a bit nonsensical, but it absolutely works.
![image](https://github.com/owid/owid-grapher/assets/2641501/4c1ebf05-ee76-4222-bfcd-eb8890082f27)

Not sure if we ever have such a chart, but since it's super easy to fix we might as well do it:
Since we use `values[0]` below, we need to use `xStart/yStart` instead so they always correspond to the same data point, I think.

```suggestion
            const xTime = xColumn.def.targetTime ?? xStart,
                yTime = yColumn.def.targetTime ?? yStart,
                hasOverrideYear = isNumber(xColumn.def.targetTime) || isNumber(yColumn.def.targetTime)
```